src/components/AddPlaidBankAccount.js

```diff
@@ -86,6 +89,13 @@ class AddPlaidBankAccount extends React.Component {
     }
 
     componentDidUpdate(prevProps) {
+        // block keyboard shortcuts that can navigate when plaid modal is shown
+        if (lodashGet(this.props.plaidData, 'bankAccounts', []).length > 0) {
+            this.unblockKeyboardShortcuts();
+        } else {
+            this.blockKeyboardShortcuts();
+        }
```

Comment:
I have some preference for subscribing the keyboard shortcuts in `componentDidMount` and setting the parameter `shouldBubble` to `() => lodashGet(this.props.plaidData, 'bankAccounts', []).length > 0` so it allows the default shortcuts to work when we have the bank accounts.

I just find it slightly easier to understand when this can happen if it is in `componentDidMount` and `componentWillUnmount` compared to `componentDidUpdate`

@0xmiroslav thoughts?